Controllers/answerController.js

```
@@ -1,5 +1,12 @@
 import database from '../services/database.js';
 import { validateAnswer } from '../validators/answerValidator.js';
+import { pubsubPublisher } from '../helpers/pubsubPublisher.js';    
+import config from '../config/prod.js';
+
+const pubsubPublisher = new pubsubPublisher(
```

Biome (1.9.4)

[error] Shouldn't redeclare 'pubsubPublisher'. Consider to delete it or rename it.

'pubsubPublisher' is defined here:


(lint/suspicious/noRedeclare)